### PR TITLE
Add support for the health and is-task filters

### PR DESCRIPTION
**Pull request by wsong**
Signed-off-by: Wayne Song <wsong@docker.com>

### CODE REVIEW

**Comment by allencloud** [comment]
Could you add some test for this? @wsong ☀️

**Comment by wsong** [comment]
Added some integration tests. Test failures appear unrelated.

**Comment by wsong** [comment]
Rebased.

File: cluster/container.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/docker/go-units"
+	units "github.com/docker/go-units"
```
Comment (nishanttotla) [inline cluster/container.go]: nit: calling it `units` separately isn't required.

**Pull request merged**